synapse/handlers/auth.py

```diff
@@ -277,6 +278,9 @@ async def check_auth(
             description: A human readable string to be displayed to the user that
                          describes the operation happening on their account.
 
+            validate_operation: Whether it should be validate that the operation
```

Comment:
Should be validate?

Reply:
We should probably also mention that if this is false, there is a side effect of updating the client dict?

Reply:
I made this a bit clearer along with the other comment about comparing URI/methods.